app/src/main/java/io/github/droidkaigi/confsched/receiver/SessionScheduleReceiver.java

```diff
@@ -0,0 +1,58 @@
+package io.github.droidkaigi.confsched.receiver;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.support.v4.app.NotificationCompat;
+import android.util.Log;
+import android.widget.Toast;
+
+import org.parceler.Parcels;
+
+import javax.inject.Inject;
+
+import io.github.droidkaigi.confsched.R;
+import io.github.droidkaigi.confsched.activity.ActivityNavigator;
+import io.github.droidkaigi.confsched.activity.MainActivity;
+import io.github.droidkaigi.confsched.activity.SessionDetailActivity;
+import io.github.droidkaigi.confsched.model.Session;
+
+public class SessionScheduleReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        Session session = Parcels.unwrap(intent.getParcelableExtra(Session.class.getSimpleName()));
+
+        // launch SessionDetailsActivity stacked with MainActivity
+        Intent sessionDetailIntent = SessionDetailActivity.createIntent(context, session);
```

Comment:
Yes, I did it.
Even if process is dead, the notification launch Detail Activity stacked on ManActivity.
Please tell me, If you find  any problem.